postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -269,13 +269,17 @@ def _get_new_pg_stat_activity(self, available_activity_columns):
         report_activity = self._report_activity_event()
         cur_time_func = ""
         blocking_func = ""
+        backend_type_predicate = "false"
```

Comment:
im confused so for versions less than 10 we have the string `false` in the query now?

Reply:
Yes, I see now that may not be ideal lol, I can re-work the query building logic.

Reply:
just set it to an empty string?